kovri.md

```diff
@@ -46,7 +46,19 @@ Let's next go over some fairly technical stuff to have a better understanding of
 
 ## 5) Kovri and Technical Attributes
 
-> **TODO**: Give a concise overview of garlic routing/encryption
+Now hold on, since we are about to overview how Kovri works.
+
+Kovri is currently built from I2P's open standards. I2P stands for the invisible internet protocol. Kovri acts as an anonymizing router as it tunnels traffic through the I2P network.
```

Comment:
>invisible internet protocol

Invisible Internet Project.

Important note: kovri doesn't "act" as an anonymizing router which uses the I2P network, kovri currently *is* the network and works alongside other implementations.